cylc/rose/entry_points.py

```diff
@@ -177,11 +176,6 @@ def record_cylc_install_options(
     """
     # Create a config based on command line options:
     cli_config = get_cli_opts_node(opts, srcdir)
-    # Leave now if there is nothing to do:
-    if not cli_config:
-        return False
```

Comment:
I think this ok, looks like `get_cli_opts_node()` always returns an object

Reply:
https://github.com/cylc/cylc-rose/pull/196/files#r1033650767